UPGRADE.md

```diff
@@ -3,6 +3,44 @@ UPGRADE
 
 # From 0.14.0 to 0.15.x
 
+### Support
+
+Contact component has been renamed to Support.
+ContactBundle has been renamed to SupportBundle.
+
+Model classes and interfaces have been renamed.
+
+* Request becomes Ticket.
+* Topic becomes Category.
```

Comment:
Maybe a more specific name like `SupportCategory` would be better. Category is very generic and can be confused.